docs/changes/4.4.3.md

```diff
@@ -5,6 +5,7 @@ Release date: `2020-xx-xx`
 ## Core
 
 - [NXDRIVE-1786](https://jira.nuxeo.com/browse/NXDRIVE-1786): Handle corrupted downloads in Direct Edit
+- [NXDRIVE-2112](https://jira.nuxeo.com/browse/NXDRIVE-2112): Always start a fresh download on Direct Edit
```

Comment:
Add the NXDRIVE-2116 too, it is the same fix and it is a good thing to reference it here.